nucypher/blockchain/eth/sol/source/contracts/PolicyManager.sol

```diff
@@ -159,9 +159,9 @@ contract PolicyManager is Upgradeable {
         Policy storage policy = policies[_policyId];
         policy.client = msg.sender;
         uint16 currentPeriod = getCurrentPeriod();
-        policy.startPeriod = currentPeriod.add16(1);
+        policy.startPeriod = currentPeriod + 1;
         policy.lastPeriod = currentPeriod.add16(_numberOfPeriods);
-        policy.rewardRate = msg.value.div(_nodes.length).sub(_firstPartialReward).div(_numberOfPeriods);
+        policy.rewardRate = msg.value.div(_nodes.length).sub(_firstPartialReward) / _numberOfPeriods;
```

Comment:
Why there's still a `div()` and `sub()` here?

Reply:
only `!=0` need to check for `div()`, `_numberOfPeriods` was checked in first `require`